### PR TITLE
Update outdated READMEs and spelling mistakes

### PR DESCRIPTION
**Pull request by weilbith**
Related to: trustlines-network/project#798

See also the new opened issue #608

### CODE REVIEW

File: tools/bridge/README-dev.md [hunk 1]
```diff
@@ -16,15 +16,14 @@ always major.minor.patch)
 
 ## bumpversion
 
-tools/bridge uses bumpversion in order to maintain it's
+`tools/bridge` uses [bumpversion](https://pypi.org/project/bumpversion/) in order
```
Comment (schmir-at-bb) [inline tools/bridge/README-dev.md]: `to maintain it's` is missing now.

File: tools/bridge/end2end-tests/README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
 # Bridge end2end tests
 
-These are the end2tests of the Trustlines Blockchain Bridge (tlbc-bridge).
-
+This are the end2end tests of the Trustlines Blockchain Bridge (tlbc-bridge).
```
Comment (schmir-at-bb) [inline tools/bridge/end2end-tests/README.md]: I'd suggest to write 'This directory contains the end2test tests...'

File: tools/bridge/README-dev.md (hunk 1 above)
Comment (weilbith) [reply tools/bridge/README-dev.md]: :heavy_check_mark:

File: tools/bridge/end2end-tests/README.md (hunk 2 above)
Comment (weilbith) [reply tools/bridge/end2end-tests/README.md]: :heavy_check_mark:

**Pull request merged**